admin-manual/installation/installation.rst

```diff
@@ -234,45 +237,71 @@ two different sources of packages.
    sudo apt-get update
    sudo apt-get upgrade
 
-4. Install the storage service package
+5. Install Elasticsearch
+
+.. code:: bash
+
+   sudo apt-get install elasticsearch
+
+6. Install the storage service package
+
+.. code:: bash
+
+   sudo apt-get install -y --force-yes archivematica-storage-service
+
+7. Install pip
+
+.. code:: bash
+
+   wget -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py
+   sudo python /tmp/get-pip.py
+
+8. Install Siegfried
 
 .. code:: bash
 
-   sudo apt-get install -y archivematica-storage-service
+   sudo apt-get install -y --force-yes siegfried
 
-5. Configure the storage service
+9. Configure the storage service
 
 .. code:: bash
 
    sudo rm -f /etc/nginx/sites-enabled/default
    sudo ln -s /etc/nginx/sites-available/storage /etc/nginx/sites-enabled/storage
-   sudo ln -s /etc/uwsgi/apps-available/storage.ini /etc/uwsgi/apps-enabled/storage.ini
-   sudo service uwsgi restart
-   sudo service nginx restart
 
-6. Install the Archivematica packages (each of these packages can be installed separately, if necessary). Say YES or OK to any prompts you get after entering the following into terminal:
+10. Install the Archivematica packages (each of these packages can be installed separately, if necessary). Say YES or OK to any prompts you get after entering the following into terminal:
 
 .. code:: bash
 
    sudo apt-get install archivematica-mcp-server
-   sudo apt-get install archivematica-mcp-client
    sudo apt-get install archivematica-dashboard
-   sudo apt-get install elasticsearch
+   sudo apt-get install archivematica-mcp-client
 
-7. Configure the dashboard
+11. Configure the dashboard
+
+.. code:: bash
+
+   sudo ln -s /etc/nginx/sites-available/dashboard.conf /etc/nginx/sites-enabled/dashboard.conf
+
+Start the Elasticsearch service and configure it to start when the system is booted.
+
+.. code:: bash
+
+   sudo /etc/init.d/elasticsearch restart
```

Comment:
This could be changed to use upstart instead?

sudo service elasticsearch restart